doc/index.rst

```diff
@@ -105,49 +107,45 @@ Workflow of the Expert System
 
 1. Preparation
 
-   1.1. Build all possible topologies. A **topology** is represented by a
-   :ref:`graph <index:1. State Transition Graphs>`, in which the edges and
-   nodes are empty (no particle information).
+  1.1. Build all possible topologies. A **topology** is represented by a
+  :ref:`graph <index:1. State Transition Graphs>`, in which the edges and nodes
+  are empty (no particle information).
 
-   1.2. Fill the topology graphs with the user provided information. Typically
-   these are the graph's ingoing edges (initial state) and outgoing edges
-   (final state).
+  1.2. Fill the topology graphs with the user provided information. Typically
+  these are the graph's ingoing edges (initial state) and outgoing edges (final
+  state).
 
 2. Solving
 
-   2.1. *Propagate* quantum number information through the complete graph while
-   respecting the specified conservation laws. Information like mass is not
-   used in this first solving step.
+  2.1. *Propagate* quantum number information through the complete graph while
+  respecting the specified conservation laws. Information like mass is not used
+  in this first solving step.
 
-   2.2. *Clone* graphs while inserting concrete matching particles for the
-   intermediate edges (mainly adds the mass variable).
+  2.2. *Clone* graphs while inserting concrete matching particles for the
+  intermediate edges (mainly adds the mass variable).
 
-   2.3. *Validate* the complete graphs, so run all conservation law check that
-   were postponed from the first step.
+  2.3. *Validate* the complete graphs, so run all conservation law check that
+  were postponed from the first step.
```

Comment:
The grey bars on the left side are not my style. If that can be changed into a nested enumeration it would be nicer.

Reply:
Ok I tried around a bit, but I can't get the bar to disappear, only with a nested **bullet** list. It seems to be a shortcoming of the Sphinx Book Theme